doc/WebSite/Entity-History.md

```diff
@@ -96,18 +96,18 @@ save change logs for a desired property.
 entity**. Thus, you can **hide sensitive data** in change logs, such as
 passwords for example.
 
-### Reason
+### Reason Property
 
-Entity change set has a **Reason** property that can be used to understand why a
-set of changes had occurred, i.e. the use case that resulted in these changes.
+The entity change set has a **Reason** property that can be used to understand why a
+set of changes have occurred, i.e. the use case that resulted in these changes.
 
 For example, Person A transfers money from Account A to Account B. Both account
-balances change and "Money transfer" is recorded as the Reason for this change set.
+balances change and the "Money transfer" is recorded as the Reason for this change set.
```

Comment:
Saying  `the "Money transfer"` places emphasis on the transfer **action**, but we mean literally `"Money transfer"` as a **string** is recorded.